pandas/core/arrays/base.py

```diff
@@ -505,6 +506,54 @@ def unique(self):
         uniques = unique(self.astype(object))
         return self._from_sequence(uniques, dtype=self.dtype)
 
+    def searchsorted(self, v, side="left", sorter=None):
```

Comment:
I personally don't like one-letter parameter names. Use ``values`` instead?

Reply:
I don't like it either, but theres value in matching NumPy here.

Reply:
Sorry, meant ``value``, as that is the name used in Series.searchsorted and various other ``searchsorted`` implementations in pandas (``Categorical.searchsorted`` at least, haven’t checked all impl.).

I think it`s inconsistent to follow numpy naming here, when pandas’s is better:-)

Reply:
we use ``value`` currently in both Index and Series. let's be consistent here (in fact I think we went thru a deprecation cycle on those a while back).

Reply:
Did you see Stephan's comment in 
https://github.com/pandas-dev/pandas/pull/24350#issuecomment-449101653?

Ideally, `np.searchsorted(extension_array)`, would always work. If we do `np.searshsorted(v=extension_array)` I think a TypeError will be raised.

Reply:
see my comment above; this would be an inconsistency in naming
which is much worse that have np.searchedsorted(ea) not working which is just convenience anyhow

Reply:
It's not just a convenience. @shoyer do you have thoughts here?

Subclasses implementing `__array_function__` will be allowed to override ExtensionArray.searchsorted with the correct function signature, but it'd be nice if things worked out of the box.